requirements.txt

```diff
@@ -2,3 +2,4 @@ anymarkup
 jsonpointer
 requests
 websocket-client
+GitPython
```

Comment:
I'm not a big fan of adding a new dependency for something that is probably going to get used 1% of the time. hmmm